packages/internal/bridge/sdk/src/lib/utils.test.ts

```diff
@@ -24,4 +33,233 @@ describe('utils', () => {
       expect(result).toEqual('devnet');
     });
   });
+
+  describe('isWithdrawNotWrappedIMX', () => {
+    it('should return false for non-wrapped IMX but not withdraw', () => {
+      const token: FungibleToken = rootIMXs.mainnet;
+      const bridgeInstance: BridgeInstance = {
+        rootChainID: '123',
+        childChainID: ETH_SEPOLIA_CHAIN_ID,
+      };
+
+      const direction: BridgeDirection = {
+        sourceChainId: bridgeInstance.childChainID,
+        destinationChainId: bridgeInstance.rootChainID,
+        action: BridgeFeeActions.DEPOSIT,
+      };
+
+      const result = isWithdrawNotWrappedIMX(token, direction, bridgeInstance);
+      expect(result).toEqual(false);
+    });
+
+    it('should return false for wrapped IMX', () => {
+      const token: FungibleToken = childWIMXs.testnet;
+      const bridgeInstance: BridgeInstance = {
+        rootChainID: '123',
+        childChainID: ETH_SEPOLIA_CHAIN_ID,
+      };
+
+      const direction: BridgeDirection = {
+        sourceChainId: bridgeInstance.childChainID,
+        destinationChainId: bridgeInstance.rootChainID,
+        action: BridgeFeeActions.WITHDRAW,
+      };
+
+      const result = isWithdrawNotWrappedIMX(token, direction, bridgeInstance);
+      expect(result).toEqual(false);
+    });
+
+    it('should return true for not wrapped IMX', () => {
+      const token: FungibleToken = rootIMXs.mainnet; // Anything that isn't a child chain WIMX.
+      const bridgeInstance: BridgeInstance = {
+        rootChainID: '123',
+        childChainID: ETH_SEPOLIA_CHAIN_ID,
```

Comment:
Should the child chain ids here and elsewhere be `ZKEVM_TESTNET_CHAIN_ID`? 
```suggestion
        childChainID: ZKEVM_TESTNET_CHAIN_ID,
```

Reply:
Given that these are just unit tests, it shouldn't make a difference how we configure the `BridgeInstance`

Reply:
Agreed. It just avoids possible confusion?